eos-tech-support/eos-usb-internet-writer

```diff
@@ -0,0 +1,229 @@
+#!/bin/bash -e
+set -o pipefail
+
+eos_write_live_image=$(dirname "$0")/eos-write-live-image
+if [ ! -f "$eos_write_live_image" ]; then
+    eos_write_live_image='eos-write-live-image'
+fi
+
+args=$(getopt -o t:h \
+    -l tmp-dir: \
+    -l debug \
+    -l help \
+    -n "$0" -- "$@")
+eval set -- "$args"
+
+tmp_dir=/tmp/eos-usb-internet-writer-tmp
+debug=
+
+usage() {
+    local self
+    self=$(basename "$0")
+    cat <<EOF
+Usage:
+    $self [options] -- IMAGE_OPTIONS
+
+Arguments:
+    IMAGE_OPTIONS       Everything after -- is passed to the $eos_write_live_image
+                        command.
+
+Options:
+    -t, --tmp-dir PATH  Temporary directory to use for downloading files and
+                        save the USB key files to copy, user is in charge of
+                        cleaning this directory if required, the command will
+                        try to all the content from this directory.
```

Comment:
Looks like part of the phrase  went missing here.

Reply:
You are right,